bridge/README.md

```diff
@@ -0,0 +1,67 @@
+# Madnet/Bridge
+
+This repository contains all solidity smart contracts used by the MadNet. 
+
+## Requirements
+* [dapp tools](https://github.com/dapphub/dapptools#installation)
+* HardHat (following)
+
+### HardHat
+
+To install hardhat and all requirements to compile and test the smart contracts in this repository,
+run the following command at the root of this repository
+
+```bash
+$ npm i -g hardhat-shorthand
+$ hardhat-completion install
+```
+
+
+## Setup
+
+### Install dependencies 
+
+Install all the necessary dependencies
+```bash
+$ npm ci
+$ npm run init 
+```
+It is mandatory to run this latest command `npm run init`, every time you run the `clean` command.  
+
+### Running unit tests
+
+You can choose between two different commands to run unit tests. The `test-parallel` will be faster
+but you won't see the gas reporter as logged once the tests are running successfully 
+`Note: Gas reporting has been skipped because plugin hardhat-gas-reporter does not support the --parallel flag.)`
+
+```bash
+$ npm run test 
+$ npm run test-parallel 
+```
+
+### Deploying
```

Comment:
Maybe we should be renaming this to 
```suggestion
### Golang code binding generation
```
We will be adding documentation about the deployment in the near future. We need to finish the hardhat scripts yet.